data-access/form-queries.ts

```diff
@@ -0,0 +1,63 @@
+import { querySudo, updateSudo } from '@lblod/mu-auth-sudo';
+import { sparqlEscapeUri } from 'mu';
+import { sparqlEscapeDateTime, sparqlEscapeString } from '../util/mu';
```

Comment:
Personally really doubting what we should actually use, thought it would be good but i catch myself using the one of the mu library more often because it aut-imports it ;p

Reply:
But when using sparqlEscapeTermValue we do use these util escapers so should be a good thing